policy/modules/system/systemd.if

```diff
@@ -1417,6 +1417,7 @@ interface(`systemd_status_networkd',`
 interface(`systemd_relabelfrom_networkd_tun_sockets',`
 	gen_require(`
 		type systemd_networkd_t;
+		class tun_socket relabelfrom;
```

Comment:
This is not necessary.

Reply:
To further clarify, `tun_socket` is a kernel object class, so it is always requried.

Reply:
Yes, I made some mistake here. Now when I checked these, I was not able to trigger the issue.